lineapy/execution/executor.py

```diff
@@ -302,6 +301,23 @@ def _execute_call(
         except ArtifactSaveException:
             # keep the error stack if its artifact save
             raise
+        except ImportError as exc:
```

Comment:
Hm I had a hard time following this logic. How do we distinguish between the user's ImportError and ours? I think this might break the user's error stack if there was an `ImportError` in a blackbox.

Reply:
good point, that might happen since we haven't added that test. An easy way out is to have a custom errors. I'll add it in next commit.